src/game.cpp

```diff
@@ -4521,21 +4522,36 @@ void Game::showPauseMenu()
 		" --> place single item to slot\n"
 		);
 #else
-	static const std::string control_text = strgettext("Default Controls:\n"
-		"- WASD: move\n"
-		"- Space: jump/climb\n"
-		"- Shift: sneak/go down\n"
-		"- Q: drop item\n"
-		"- I: inventory\n"
+	static const std::string control_text_template = strgettext("Controls:\n"
+		"- %s%s%s%s: move\n"
+		"- %s: jump/climb\n"
+		"- %s: sneak/go down\n"
+		"- %s: drop item\n"
+		"- %s: inventory\n"
 		"- Mouse: turn/look\n"
 		"- Mouse left: dig/punch\n"
 		"- Mouse right: place/use\n"
 		"- Mouse wheel: select item\n"
-		"- T: chat\n"
+		"- %s: chat\n"
 	);
+
+	 char control_text[500];
+
+	 snprintf(control_text, 500, control_text_template.c_str(),
+			 GET_KEY_NAME(keymap_forward),
```

Comment:
it will just look a bit strange as instead of a key there will be an empty string. That's an edge case anyway that would require the user to manually unbind the key in `minetest.conf`